README.md

```diff
@@ -3,19 +3,33 @@
 ## Installation with pip
 
 1. Install the library using pip (currently the library is available under pypi test repository):  
-```pip install --index-url https://pypi.org/simple --extra-index-url https://test.pypi.org/simple/ kentik-api```
+```pip3 install --index-url https://pypi.org/simple --extra-index-url https://test.pypi.org/simple/ kentik-api```
 1. Check installation successful - no errors should be reported:  
-```python -c "import kentik_api"``` 
-1. Run an example (Optional):
+```python3 -c "import kentik_api"```
+1. Run an example (optional):
   ```bash
   export KTAPI_AUTH_EMAIL=<your kentik api credentials email>
   export KTAPI_AUTH_TOKEN=<your kentik api credentials token>
-  python kentik_api_library/examples/sites_example.py
+  python3 kentik_api_library/examples/sites_example.py
   ```
 
 ## Usage examples
 
-For usage examples please see: kentik_api_library/examples/
+For library usage examples please see: kentik_api_library/examples/  
+List of available examples:
+- alerting_example.py - create Manual Mitigation
+- applications_example.py - create/update/delete Custom Application
+- bulk_user_create.py - create users from YAML file
+- devices_example.py - create/update/get/delete/list Devices
+- dimensions_example.py - create/update/get/delete/list Custom Dimensions, create/update/delete Populator
+- labels_examply.py - create/update/get/delete/list Device Labels
```

Comment:
```suggestion
- [labels_example.py](./kentik_api_library/examples/labels_example.py) - create/update/get/delete/list Device Labels
```
I would use relative path link, so that IDEs detect typos/renames such as that one

Reply:
let's discuss it with Maciej and do in separate PR